.storybook/webpack.config.js

```diff
@@ -1,4 +1,5 @@
 const path = require( 'path' );
+const MiniCssExtractPlugin = require( 'mini-css-extract-plugin' );
 
 module.exports = async ( { config } ) => {
 	config.resolve = {
```

Comment:
@tofumatt can we update this to resolve `@wordpress/api-fetch` with an alternate version just for Storybook which returns mocked data since the below will no longer work?

https://github.com/google/site-kit-wp/blob/7c192a71c09610e95d2d5fbcefad16b7abbe9d71/.storybook/config.js#L89-L111